tests/test_cli_sources.py

```diff
@@ -49,6 +49,34 @@ def side_effect(fields):
     )
 
 
+@pytest.mark.usefixtures("token_environ")
+@mock.patch("mapbox_tilesets.scripts.cli.MultipartEncoder")
+@mock.patch("mapbox_tilesets.scripts.cli.MultipartEncoderMonitor")
+@mock.patch("requests.Session.post")
+def test_cli_add_source_wrong_username(
+    mock_request_post,
+    mock_multipart_encoder_monitor,
+    mock_multipart_encoder,
+    MockResponse,
+    MockMultipartEncoding,
+):
+    if "MAPBOX_ACCESS_TOKEN" in os.environ:
+        del os.environ["MAPBOX_ACCESS_TOKEN"]
+
+    os.environ["MapboxAccessToken"] = "pk.eyJ1Ijoid3JvbmctdXNlciJ9Cg.xxx"
```

Comment:
Could you add a comment here on who is the owner for that token and where are the credentials stored?

Reply:
OK, comment added. This is the base64 encoding of `{"u":"wrong-user"}`, not a real token.